deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```diff
@@ -582,7 +606,17 @@ service:
         - fluentforward
 {{ end }}
 {{ end }}
-
+## Use this pipeline for cloudwatch logs
+{{- if .Values.sumologic.logs.collector.otelcloudwatch.enabled }}
+    logs/otlp/otelcloudwatch:
+      receivers:
+        - otlp
+      processors:
+        - memory_limiter
+        - batch
+      exporters:
+        - sumologic/systemd
```

Comment:
I think we will want to send it to `containers` endpoint (and to metadata statefulset - as I understand, you haven't debugged that one yet)

Reply:
That part works now, the logs are being sent to sumo logic via the metadata statefulset.
https://stagdata.long.sumologic.net/ui/#/search/create?id=EmilVfSu90JlAmXg2EkjawD3nsDpWabmADKiLRWF

Yes I can change it to use the containers endpoint

Reply:
Nevermind then, I did not notice that this is the configmap for metadata otelcol.